components/d2l-filter-dropdown/d2l-filter-dropdown-page.js

```diff
@@ -96,19 +95,19 @@ class D2LFilterDropdownPage extends mixinBehaviors([D2L.PolymerBehaviors.FilterD
 	}
 
 	_handleSearchChange(e) {
-		var value = e.detail.value;
-		var clear = value === '';
-		if (clear || value.length) {
-			for (var i = 0; i < this.options.length; i++) {
-				this._setOptionProp('display', clear || this._caseInsensitiveContainsSubstring(this.options[i].title, value), i);
-			}
-		}
-	}
-
-	_caseInsensitiveContainsSubstring(s, sub) {
-		const lowerS = s.toLowerCase();
-		const lowerSub = sub.toLowerCase();
-		return lowerS.indexOf(lowerSub) >= 0;
+		this.dispatchEvent(
+			new CustomEvent(
+				'd2l-filter-dropdown-page-searched',
```

Comment:
An interesting thing that could happen now - if we have 2 filter components on the same page, and they both use the same filter category (like might happen to us with the grouped activity view) then searching one will also search the other (but the search text won't be in the other search bar)